apps/dashboard/src/utils/pricing.tsx

```diff
@@ -1,20 +1,24 @@
-import { type AccountPlan, accountPlan } from "@3rdweb-sdk/react/hooks/useApi";
 import Link from "next/link";
+import type { Team } from "../@/api/team";
 
 export const CONTACT_US_URL =
   "https://meetings.hubspot.com/sales-thirdweb/thirdweb-pro";
 
-export const PLANS: {
-  [T in AccountPlan]: {
+type NonFreeTeamPlan = Exclude<Team["billingPlan"], "free">;
+
+// Question: @Jonas is this correct?
+export const TEAM_PLANS: Record<
```

Comment:
@jnsdls question for you. Is the features listed on each plan still correct?

Reply:
for now lets say yes